apps/pie-storybook/decorators/component-status.ts

```diff
@@ -26,7 +26,7 @@ export const ComponentStatus = (story: () => TemplateResult, storybookContext: S
         <div class="pie-sb-status-wrapper">
             ${story()}
 
-            <div class="${`c-componentStatus ${positionClass}`}">
+            <div class="${`c-componentStatus ${positionClass} percyHidden`}">
```

Comment:
this removes the status from both storybook deployments

Reply:
Fixed! Forgot the Percy media query 🤦🏼